bridges/primitives/chain-asset-hub-rococo/src/lib.rs

```diff
@@ -46,6 +46,10 @@ frame_support::parameter_types! {
 	/// Base delivery fee to `BridgeHubRococo`.
 	/// (initially was calculated by test `BridgeHubRococo::can_calculate_weight_for_paid_export_message_with_reserve_transfer`)
 	pub const BridgeHubRococoBaseFeeInRocs: u128 = 1214739988;
+
+	/// User fee for ERC20 token transfer back to Ethereum.
+	/// (initially was calculated by test `OutboundQueue::calculate_fees` - ETH/ROC 1/411 and fee_per_gas 15 GWEI = 18679250000 + *25%)
```

Comment:
The fee parameters should result in a fee that is greater or equal to the fee generated by these parameters:

https://github.com/Snowfork/snowbridge/blob/2be6f57a2bc2c0e162cda32c331dafe5b80a8b80/parachain/pallets/outbound-queue/src/lib.rs#L252

So its a problem if fee_per_gas is 15gwei here and 30gwei on bridgehub.

Reply:
I didn't realize those values in https://github.com/Snowfork/snowbridge/blob/2be6f57a2bc2c0e162cda32c331dafe5b80a8b80/parachain/pallets/outbound-queue/src/lib.rs#L252 are the final values I should use. I wrote a script that calculates the average gas price for the last three weeks like you suggested, and it was around 15 gwei. I'll change the calculation to use the values defined in the outbound queue then.

Reply:
Ah, rather update the values in the outbound queue, as your figure is more accurate :)

Reply:
To clarify, I wasn't saying those values were final, but rather that if we update the fees to be more accurate in one place, we must reconcile the fees in other places too.

Reply:
Got it, agreed. Will add a comment about it too.